packages/graphile-build/__tests__/enum.test.js

```diff
@@ -1,7 +1,16 @@
-const { GraphQLEnumType } = require("graphql");
-const { printSchema } = require("graphql/utilities");
+const { GraphQLEnumType, isSchema, printSchema } = require("graphql");
 const { buildSchema, defaultPlugins } = require("../");
 
+const GraphQLSchemaSerializer = {
+  test(val) {
+    return isSchema(val);
+  },
+  serialize(schema) {
+    return printSchema(schema);
+  },
+};
+expect.addSnapshotSerializer(GraphQLSchemaSerializer);
```

Comment:
Please can you put this in `packages/graphile-build/__tests__/helpers.js` and then import that instead:

```js
require("./helpers");
```

Reply:
Given this is used throughout, actually putting it in a helpers file that everyone can use would be awesome. Maybe we should add a private subpackage just for this so we can require it from the other projects?

Reply:
I think a different package is probably the right solution here, but I'm not exactly sure what you mean by "private subpackage". Should I make a new directory under `packages` in this repo? How do I make it private? Shouldn't the versioning be independent of everything else in the `packages` directory? I've never used Lerna before, and I'm not sure how to configure it properly to make this work...